src/test/java/com/cst438/controller/SectionControllerUnitTest.java

```diff
@@ -42,6 +42,7 @@ public void addSection() throws Exception {
                 2024,
                 "Spring",
                 "cst499",
+                "Title",
```

Comment:
I know I asked you to remove my changes, but I think these here are needed for these unit tests to work.